apps/common/send_email.py

```diff
@@ -0,0 +1,40 @@
+from fastapi_mail import FastMail, MessageSchema, ConnectionConfig
+from settings import Settings
+from typing import List
+
+
+conf = ConnectionConfig(
+    MAIL_USERNAME=Settings.MAIL_USERNAME,
+    MAIL_PASSWORD=Settings.MAIL_PASSWORD,
+    MAIL_FROM=Settings.MAIL_FROM,
+    MAIL_PORT=Settings.MAIL_PORT,
+    MAIL_SERVER=Settings.MAIL_SERVER,
+    MAIL_FROM_NAME=Settings.MAIL_FROM_NAME,
+    MAIL_TLS=Settings.MAIL_TLS,
+    MAIL_SSL=Settings.MAIL_SSL,
+    USE_CREDENTIALS=Settings.USE_CREDENTIALS,
+    TEMPLATE_FOLDER=Settings.TEMPLATE_FOLDER,
+)
+
+
+async def send_email_async(
```

Comment:
@victor-nuzhniy i wrote in Jira ticket "create interface". so i mean that u need to create abstract class with base methods, after create class for current implementation of email sending, etc.
pls check how implemented file "file_managers.py" and make the same arch

Reply:
also pls rename the current file to "email_sender_managers.py"

Reply:
ok